code/src/WijmoProvider/Features/Selection.ts

```diff
@@ -318,15 +318,30 @@ namespace WijmoProvider.Feature {
                 ...this._grid.provider.selectedRanges.filter((p) => p.isValid)
             );
             // }
-            return this._getCheckedRows()
+
+            // create checkedRows cell range.
+            let checkedRowsRange = this._getCheckedRows()
```

Comment:
The name could be different as checkedRowsRange sounds like you are only going to have the rows that have been checked ☑️.